sw/device/tests/sim_dv/all_escalation_resets_test.c

```diff
@@ -323,11 +324,12 @@ static void lc_ctrl_fault_checker(bool enable, const char *ip_inst,
       .mask = UINT32_MAX, .index = kDifLcCtrlStatusCodeTooManyTransitions};
   uint32_t mask = bitfield_field32_write(0, relevant_field, UINT32_MAX);
   uint32_t relevant_status = status & mask;
+  uint32_t bus_integ_error =
+      bitfield_bit32_write(0, kDifLcCtrlStatusCodeBusIntegError, true);
+  uint32_t state_error =
+      bitfield_bit32_write(0, kDifLcCtrlStatusCodeCorrupt, true);
   uint32_t expected_status =
-      enable
-          ? (bitfield_bit32_write(0, kDifLcCtrlStatusCodeBusIntegError, true) &
-             mask)
-          : 0;
+      enable ? (type == we_check ? bus_integ_error : state_error) : 0;
```

Comment:
This looks like an important fix. Does it make sense to put it in a separate commit with its own commit message?

Reply:
Well, this is not really a separate fix, but it is required to handle one of the two added alerts.